tests/test_wave_05.py

```diff
@@ -80,34 +81,46 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+#@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    #raise Exception("Complete test")
     # Act
     # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json={"title": "Updated goal 1"})
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
     # assertion 1 goes here
+    assert response.status_code == 200
     # assertion 2 goes here
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "Updated goal 1"
+        }
+    }
     # assertion 3 goes here
+    assert "goal" in response_body
```

Comment:
Note that it is redundant to have both of these asserts since the first assert covers the second one. I'd recommend only having the first assert (below) since it is more specific.
```
assert response_body == {
    "goal": {
        "id": 1,
        "title": "Updated Goal Title"
    }
}
```